olive/passes/onnx/conversion.py

```diff
@@ -75,8 +85,8 @@ def _run_for_config(
 
         # TODO: add e2e test for model on cpu but data on gpu; model on gpu but data on cpu
         # put pytorch_model and dummy_inputs at the same device
-        pytorch_model.to("cpu")
-        dummy_inputs = tensor_data_to_device(dummy_inputs, "cpu")
+        pytorch_model.to(device)
+        dummy_inputs = tensor_data_to_device(dummy_inputs, device)
```

Comment:
Can you move the pytorch model back to cpu after the conversion? So that it does not keep using gpu memory after the pass is done.

Related comment on the other PR https://github.com/microsoft/Olive/pull/478#discussion_r1290704153

Reply:
done